### PR TITLE
Fix glance image hosting in HA cluster

### PR DESCRIPTION
**Pull request by dirkmueller**
The images need to be on shared storage.

### CODE REVIEW

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-3a-kvm-hyperv.yaml [hunk 1]
```diff
@@ -36,6 +36,24 @@ proposals:
       - "@@controller2@@"
       - "@@controller3@@"
 
+proposals:
+- barclamp: nfs_client
+  name: glance
+  attributes:
+    exports:
+      images:
+        nfs_server: 10.162.26.129
```
Comment (vuntz) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-3a-kvm-hyperv.yaml]: Is it the same IP address for all qa clouds?

**Comment by bmwiedemann** [comment]
cant we make this DRY? 
who empties the shared glance dir so that it does not overflow?
otherwise +1

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-3a-kvm-hyperv.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-3a-kvm-hyperv.yaml]: currently yes.. we can make it configurable at some point later..

**Comment by dirkmueller** [comment]
@bmwiedemann there is a card in team green for cleaning up the scenarios. we can make it DRY by not checking in the yaml files but use e.g. yaml.erb to expand the stuff. but thats beyond the scope of getting scenarios to run right now..

**Pull request merged**